formats/cbor/commonMain/src/kotlinx/serialization/cbor/internal/Encoding.kt

```diff
@@ -227,11 +432,13 @@ internal open class CborReader(private val cbor: Cbor, protected val decoder: Cb
     override val serializersModule: SerializersModule
         get() = cbor.serializersModule
 
-    protected open fun skipBeginToken() = setSize(decoder.startMap())
+    protected open fun skipBeginToken() = setSize(decoder.startMap(tags))
```

Comment:
`skipBeginToken` is called at the end of `beginStructire`, and I do not see any tags assigned there, so I guess here it is always `null`, which looks suspicious.

Reply:
you are correct!